Libs/MRML/Core/vtkMRMLStorageNode.cxx

```diff
@@ -144,6 +144,8 @@ void vtkMRMLStorageNode::WriteXML(ostream& of, int nIndent)
       else
         {
         vtkWarningMacro("WriteXML: unable to convert relative file path to absolute, still using " << this->FileName);
+        GetVTKMessageLog()->SetMessage(vtkMessageLog::Message(vtkCommand::WarningEvent,
```

Comment:
Looks like these entries in the message log are always the same as the corresponding macro text.  In this case I'd prefer to see something like `vtkMRMLStorageNodeWarningMacro` that internally uses `vtkWarningMacro` and sets the message log.  Same for error cases.

Reply:
The current text for these SetMessage calls is just a placeholder.  We should definitely change these.  The brevity level, format, etc. for these user-friendly messages will become more obvious (to me) once I start seeing them in the dialog box in the upcoming implementation.

Reply:
+1 for `AddMessage`